docs/maui/converters/double-to-int-converter.md

```diff
@@ -0,0 +1,102 @@
+---
+title: .NET MAUI DoubleToIntConverter
+author: bijington
+description: "The DoubleToIntConverter is a converter that allows users to convert an incoming `double` value to an `int` and vice-versa."
```

Comment:
Where does this description show up? Does it make sense to have backticks for markup in here?

Reply:
That's a very good question! I don't know the answer actually. I can remove those as I doubt it works.

Reply:
Yeah I assume it shows up as the summary in search result and such. Also check the other PRs for this, I noticed it with more. You could double-check with @davidbritch

Reply:
The Title and Description metadata is used purely for SEO purposes. There's no point using backpacks in any of the metadata (anything between --- and ---).

Reply:
Would that be something to include in the tooling to give warnings about that?

Reply:
It could be, but a tooling update request succeeding is a rare event.